server.js

```diff
@@ -30,6 +30,16 @@ if ('development' === app.get('env')) {
   app.use(errorhandler());
 }
 
+var opn = require('opn');	//to open the url
+// opens the url in the default browser
 http.createServer(app).listen(app.get('port'), function () {
     console.log('Server listening on port ' + app.get('port'));
+	
+	setTimeout(
+		function(){
+			opn('http://'+'localhost:'+(process.env.PORT || 3000)+'/');			//open the url, after running server
+		},
+		1000
+	);
```

Comment:
Open waves explorer on, after 1 second, after running the server. Open it on localhost, with default or specified port.